Generic_Plugin_WidgetForum.php

```diff
@@ -119,7 +119,7 @@ function action_widget_latest_ajax() {
 	 * @return void
 	 */
 	function widget_latest_control( $widget_id, $form_inputs = array() ) {
-		if ( $_SERVER['REQUEST_METHOD'] == 'POST' ) {
+		if ( 'POST' === isset( $_SERVER['REQUEST_METHOD'] ) ? sanitize_text_field( wp_unslash( $_SERVER['REQUEST_METHOD'] ) ) : '' ) {
```

Comment:
Fix logic.  This is comparing `'POST'` to the return of `isset()`, which is always false.  Just added parentheses to encapsulate the ternary operation.
```
if ( 'POST' === ( isset( $_SERVER['REQUEST_METHOD'] ) ? sanitize_text_field( wp_unslash( $_SERVER['REQUEST_METHOD'] ) ) : '' ) ) {
```

Reply:
I don't follow why this is really needed as it's a simple conditional check and the return of isset in this case isn't always false. Either way I've added the parenthesis.